lab2.md

```diff
@@ -142,10 +142,41 @@ $ docker service create -d \
 stefanprodan/faas-grafana:4.6.3
 ```
 
+> Note: If you're running on Kubernetes, deploy Grafana with the following commands:
+>
+> Run Grafana in OpenFaaS Kubernetes namespace:
+> ```
+> kubectl -n openfaas run \
+> --image=stefanprodan/faas-grafana:4.6.3 \
+> --port=3000 \
+> grafana
+> ```
+>
+> Expose Grafana with a NodePort:
+> ```
+> kubectl -n openfaas expose deployment grafana \
+> --type=NodePort \
+> --name=grafana
+> ```
+>
+> Find Grafana node port address:
+> ```
+> $ GRAFANA_PORT=$(kubectl -n openfaas get svc grafana -o jsonpath="{.spec.ports[0].nodePort}")
+> $ GRAFANA_URL=http://$(minikube ip):$GRAFANA_PORT/dashboard/db/openfaas
+> ```
+>
+> Or you may run this port-forwarding command in order to be able to access Grafana on `http://127.0.0.1:3000`:
+> ```
+> $ POD=$(kubectl get pods -n openfaas -l app=grafana -o jsonpath="{.items[0].metadata['name']}")
```

Comment:
This command fails because the pod doesn't have a `app=grafana` label. `run=grafana` does exist though. I'm not sure what the best way is to find the pod name.